testsuite/shared/src/main/java/org/jboss/as/test/integration/domain/management/util/DomainLifecycleUtil.java

```diff
@@ -149,6 +149,13 @@ public void start() {
                         .addProcessControllerJavaOptions(javaOpts);
             }
 
+            if (configuration.isEnableAssertions()) {
+                commandBuilder.addProcessControllerJavaOption("-ea");
+                commandBuilder.addHostControllerJavaOption("-ea");
+                // This doesn't work; the HC Main class treats this as an invalid param
+                //commandBuilder.addServerArgument("-ea");
```

Comment:
Not really, no. This ends up as a command line param to the HC Main class and it doesn't know what to do with it. It only knows how to handle specific things.

The way to control server jvm settings is via the <jvm> elements in domain/host.xml, which I update in one of the commits here. For this command line thing to work, the HC would have to understand "-ea" as meaning to add that to each JVM launch. Not worth implementing IMHO.